pkg/registry/testdata/overwrite/prometheus.0.14.0/manifests/servicemonitor.crd.yaml

```diff
@@ -0,0 +1,224 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
```

Comment:
we don't need CRDs and dependencies to ensure that overwriting works, right?

Reply:
We may want to test those cases too